### DISCUSSION TITLE
Accordion problem in my project

### DISCUSSION THREAD

**Discussion by LaurentRek** [Orange-OpenSource/Orange-Boosted-Bootstrap#2782] [Q&A] [answered]
Hello everyone,
I can't get the accordion to work properly. When I click on the arrow the accordion doesn't close.
I should point out that I'm in an angular project, i've upgraded to 5.3.3 and popperJs is installed.
The problem was already present with boosted 5.3.2.
Need to install another package?
Thanks in advance

"@popperjs/core": "^2.11.8",
"boosted": "5.3.3",

![image](https://github.com/user-attachments/assets/7ab109d1-d476-4ed3-b609-6dcb6203ca04)

**Comment by louismaximepiton** [comment]
Hello Laurent,

Could you provide a reproducible example using [stackblitz](https://stackblitz.com/) for example ? If not, maybe you can share your `angular.json` and `package.json` (not the confidential parts) ?
From what you described above it seems like you have imported twice the js. It can be by redundancy or if you are for example using `ng-bootstrap` and Boosted js. Feel free to take a look at some internal examples (which I won't share here) to see how they are implemented.

**Comment by LaurentRek** [comment, marked as answer]
With your question you've fix my problem.
In fact the boosted bundle was add in the scripts part of the angular.json file. I removed and it work 
"scripts": ["./node_modules/quill/dist/quill.js", "./node_modules/boosted/dist/js/boosted.bundle.min.js"]

Thank you so much
